src/common/tools/validate.ts

```diff
@@ -14,23 +14,40 @@ For each suite in SUITE_DIRS, validate some properties about the file:
 - That each case query is not too long
 
 Example:
-  tools/validate src/unittests/ src/webgpu/
+  tools/validate src/unittests src/webgpu
 
 Options:
-  --help          Print this message and exit.
+  --help           Print this message and exit.
+  --validate-meta  Validate listing_meta.json files.
```

Comment:
The arg checked for on line 37 is `--print-metadata-warnings`

Reply:
Oops, I renamed it and forgot to fully rename it.